source/config/Config.cpp

```diff
@@ -2864,7 +2872,9 @@ bool Config::ParseConfigFile(const string &file, ConfigFileType configFileType)
         }
     }
 
+#if !defined(DISABLE_MQTT)
     LOGM_INFO(TAG, "Successfully fetched JSON config file: %s", Sanitize(contents).c_str());
```

Comment:
what is the reasoning behind checking the flag here?

Reply:
each line is printed as error message in GG component.